tools/sharkbait/src/sharkbait/permissions.clj

```diff
@@ -21,9 +22,10 @@
   "Sets the allowed actions for a permission definition."
   [permission-def]
   (let [delegate (.getAttributeDefActionDelegate permission-def)]
-    (.configureActionList delegate ["read" "write" "own"])
-    (add-implied-action delegate "own" "write")
-    (add-implied-action delegate "write" "read"))
+    (.configureActionList delegate [read write own originator])
+    (add-implied-action delegate originator own)
```

Comment:
Yeah, this isn't the permission level for editing the labels of public apps; that one's still coming. The purpose of this level is to allow us to distinguish apps that the user integrated from apps to which the user has been given `own` permissions. We _could_ do that by filtering out apps in the user's `Apps Under Development` category, but that will stop working when we do away with the current category system. This seemed like a more sustainable option.